hax/test/integration/test_motr.py

```diff
@@ -1007,3 +1007,169 @@ def my_get(key: str, recurse: bool = False, **kwds):
     assert AssertionPlan(tr_and(tr_method('ha_broadcast'),
                                 node_fid_failed())).not_exists(traces), \
         'Node failure should not be broadcast'
+
+
+def _generate_sub_disks() -> List[HaNoteStruct]:
+    drive_ha_notes: List[HaNoteStruct] = []
+    for fkey in range(2196):
+        drive_fid = create_drive_fid(fkey)
+        drive_ha_notes.append(HaNoteStruct(no_id=drive_fid.to_c(),
+                                           no_state=HaNoteStruct.M0_NC_ONLINE))
+    print(str(drive_ha_notes))
+    return drive_ha_notes
+
+
+def test_broadcast_more_than_1024_objects(mocker, planner, motr, consumer,
```

Comment:
I'm sorry, I don't quite get how this test generates more than 1024 objects to broadcast. Can you highlight that in the code please?

Reply:
```
 mocker.patch.object(motr,
                        '_generate_sub_disks',
                        return_value=_generate_sub_disks())
```

```
def _generate_sub_disks() -> List[HaNoteStruct]:
    drive_ha_notes: List[HaNoteStruct] = []
    for fkey in range(2196):
        drive_fid = create_drive_fid(fkey)
        drive_ha_notes.append(HaNoteStruct(no_id=drive_fid.to_c(),
                                           no_state=HaNoteStruct.M0_NC_ONLINE))
    print(str(drive_ha_notes))
    return drive_ha_notes
```